src/data/faq_de.json

```diff
@@ -325,6 +325,14 @@
+                            "iOS Nutzer, die bisher bspw. iOS 13.5 genutzt haben, müssen Ihr iOS Betriebsystem nun zwingend auf mindestens iOS 13.7 aktualisieren, um die CWA weiterhin zu nutzen. Nur in diesem Fall kann eine technisch korrekte Funktion der App sichergestellt werden.",       
+                            "<ul><li>Bitte führen Sie zunächst ein Update Ihrer iOS-Version durch. Auch ältere Smartphones, wie bspw. das iPhone SE, 6S oder 6S Plus unterstützen das neueste iOS Betriebssystem. Sie können Ihr iPhone somit mindestens auf iOS 13.7 aktualisieren. Aus Sicherheitsgründen sollten Sie ihr iPhone immer aktuell halten, da Apple auch regelmäßig kleinere Sicherheitslücken mit den Updates behebt. Sie können dies auf Ihrem iPhone unter Einstellungen – Allgemein – Softwareupdate durchführen.</li><li>Im Anschluss aktualisieren Sie bitte die Corona-Warn-App. Die Corona-Warn-App muss zudem mindestens in Version 1.5 installiert sein. Inzwischen ist bereits Version 1.11 verfügbar. Wir bitte Sie daher, Ihre Corona-Warn-App zu aktualisieren.</li> </ul>",
+                            "Ab der CWA Version 1.12 werden zusätzlich auch die ältere iPhone Modelle iPhone 5s, iPhone 6, iPhone 6plus unterstützt. Diese Geräte müssen auf die iOS Version 12.5 bzw. 12.5.1 aktualisiert werden",
+                            "Die folgende Tabelle dient zur Übersicht:",
+                            "<table class='table'><tr><th>Gerät</th><th>Betriebssytem<br>Minimum</th><th>Betriebssytem<br>Empfohlen</th><th>Exposure Notification System</th><th>CWA<br>Minimum</th><th>CWA<br>Empfohlen</th></tr><tr><td>iPhone 5s<br>iPhone 6<br>iPhone 6 Plus</td><td>iOS 12.5</td><td>iOS 12.5.1</td><td>v2 ENF (inbegriffen in iOS)</td><td>Zukünftige<br>Version<br>1.12</td><td>Zukünftige<br>Version<br>1.12</td></tr><tr><td>iPhone SE<br>iPhone 6s<br>iPhone 6s Plus<br>und neuere Modelle</td><td>iOS 13.7</td><td>iOS 14.4</td><td>v2 ENF (inbegriffen in iOS)</td><td>1.5</td><td>1.11</td></tr><tr><td>Android</td><td>Android 6</td><td>neuste Version vom Gerätehersteller</td><td>aktuelle Google Play services</td><td>1.5</td><td>1.11</td></tr></table><hr/>",
+                            "Siehe auch <a href='#no_risk_update' target='_blank' rel='noopener noreferrer'>[Apple/iOS]: Warum ist iOS 13.7 Mindestvoraussetzung?</a>",
+                            "und <a href='https://www.coronawarn.app/de/blog/2020-12-15-ios-12-5' target='_blank' rel='noopener noreferrer'>Telekom und SAP werden die Corona Warn App für iOS 12.5 verfügbar machen.</a>",
                             "<b>AKTUELL</b><br/><a href='https://www.coronawarn.app/de/blog/2020-12-15-ios-12-5' target='_blank' rel='noopener noreferrer'>Telekom und SAP werden die Corona Warn App für iOS 12.5 verfügbar machen.</a><hr/>",

```

Comment:
AKTUELL => AKTUELL 4. Februar 2021
since there is another AKTUELL at the top of this article "AKTUELL 8. Februar 2021"

Reply:
@MikeMcC399 I removed the previous "AKTUELL" it since it was only the link to the blog, so no historic information is lost in this case.